npm/react18/src/index.ts

```diff
@@ -25,6 +25,26 @@ const cleanup = () => {
   return false
 }
 
+/**
+ * Mounts a React component into the DOM.
+ * @param jsx {import('react').JSX.Element} The React component to mount.
+ * @param options {MountOptions} Options to pass to the mount function.
+ * @param rerenderKey {string} A key to use to force a rerender.
```

Comment:
```suggestion
 * @param {import('react').JSX.Element} jsx The React component to mount.
 * @param {MountOptions} options Options to pass to the mount function.
 * @param {string} rerenderKey A key to use to force a rerender.
```
Types before param name